DirectProgramming/DPC++FPGA/Tutorials/Tools/experimental/platform-designer/build_design.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -x
```

Comment:
This is explained step by step in the README so I think it should be removed

Reply:
will move to regtest, it's just easier to keep it here for development